### ISSUE TITLE
Keyboard focus does not work with <area is="map-area">

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#695]
If you tab to focus the map, with scripting enabled, the next tab will take you to the zoom in control, bypassing the \<area is="map-area"> elements; if you disable scripting you tab to the area elements in sequence, with the focus indicator showing where you are.

https://maps4html.org/experiments/progressive-enhancement/

**Comment by Malvoz** [comment]
> bypassing the &lt;area is="map-area"> elements

Note that when scripting is enabled `<area>` and `<img usemap>` are themselves not actually active (since [`05aadb1`](https://github.com/Maps4HTML/Web-Map-Custom-Element/commit/05aadb1b3bdcf2af406520a9dcc2fdca8edc28a5#diff-b39c0806c702c0d0b42eee8f14c8213e39e97a93df4d86e722cf98fa133f5d52R143-R164)). I think [that's still expected](https://github.com/Maps4HTML/MapML/issues/191#issuecomment-864387340).

The rendered SVG should essentially be [feature links](https://maps4html.org/experiments/linking/features/) though?
